django_ibmi/features.py

```diff
@@ -0,0 +1,42 @@
+from django.db.backends.base.features import BaseDatabaseFeatures
```

Comment:
We're no longer trying django.db.backends. Is that a pre Django 2 package?

Reply:
copying the way it is done by all the backends here https://github.com/django/django/tree/stable/3.0.x/django/db/backends

Reply:
Ok, makes sense